linux_os/guide/system/bootloader-grub2/file_groupowner_efi_grub2_cfg/rule.yml

```diff
@@ -30,6 +37,14 @@ references:
     iso27001-2013: A.10.1.1,A.11.1.4,A.11.1.5,A.11.2.1,A.13.1.1,A.13.1.3,A.13.2.1,A.13.2.3,A.13.2.4,A.14.1.2,A.14.1.3,A.6.1.2,A.7.1.1,A.7.1.2,A.7.3.1,A.8.2.2,A.8.2.3,A.9.1.1,A.9.1.2,A.9.2.3,A.9.4.1,A.9.4.4,A.9.4.5
     cis-csc: 12,13,14,15,16,18,3,5
 
+{{% if product == "fedora" %}}
+ocil_clause: '{{{ ocil_clause_file_group_owner(file="/boot/efi/EFI/fedora/grub.cfg", group="root") }}}'
+{{% else %}}
 ocil_clause: '{{{ ocil_clause_file_group_owner(file="/boot/efi/EFI/redhat/grub.cfg", group="root") }}}'
+{{% endif %}}
 
+{{% if product == "fedora" %}}
+ocil: '{{{ ocil_file_group_owner(file="/boot/efi/EFI/fedora/grub.cfg", group="root") }}}'
+{{% else %}}
 ocil: '{{{ ocil_file_group_owner(file="/boot/efi/EFI/redhat/grub.cfg", group="root") }}}'
+{{% endif %}}
```

Comment:
@iokomin please don't wrap `ocil` or `ocil_clause` YAML keys in Jinja. Please fix.